### DISCUSSION TITLE
mosaic data augmentation

### DISCUSSION THREAD

**Discussion by alicera** [open-mmlab/mmpose#1241] [Ideas]
Do you use the mosaic data augment ?
If not, is it a good idea to add it when training ?
https://github.com/ultralytics/yolov5/blob/e7bf38277f57086bf37486201909e9c04acfaa48/utils/datasets.py#L683
https://github.com/arijitgupta42/YOLO-Mosaic

**Comment by jin-s13** [comment]
I have not tried this data augmentation. 
I think it worth trying.
